packages/peregrine/lib/talons/AccountMenu/useAccountMenuItems.js

```diff
@@ -48,7 +48,7 @@ export const useAccountMenuItems = props => {
         {
             name: 'Account Information',
             id: 'accountMenu.accountInfoLink',
-            url: ''
+            url: 'account-information'
```

Comment:
Do you need a preceding `/`? Seems like the route works without but I'd add it since the other url routes in this file have one.

Reply:
```suggestion
            url: '/account-information'
```